sample/src/main/java/com/metova/privvy/sample/MainActivity.java

```diff
@@ -18,6 +24,25 @@ protected void onCreate(Bundle savedInstanceState) {
                 .hostModule(new HostModule(this))
                 .build();
 
+        hostDelegate = new PrivvyHostDelegate(this);
+
         setContentView(R.layout.main_activity);
+
+        initialize(NumberComponent.ROUTE_DATA, ButtonComponent.ROUTE_DATA);
+    }
+
+    @Override
+    public void initialize(RouteData... routeData) {
```

Comment:
Is any of this code going to change? It seems like we could get rid of this boilerplate somehow.